### DISCUSSION TITLE
Better docs pages order to not scare away newcomers

### DISCUSSION THREAD

**Discussion by NawfelBengherbiaSfeir** [TanStack/query#6891] [General] [answered]
Hello,

In the docs webpage, could you move the sidebar sections "API Reference" and "ESLint" under "Getting started" and "Guide" please?

I almost did not find the Guide pages. I think they are not very discoverable. Having beginners face the API reference as the first thing they see makes React query not very approachable.

Thanks in advance

**Comment by TkDodo** [comment]
yes, we're on it.

**Reply by TkDodo** [reply, marked as answer]
see: https://github.com/TanStack/tanstack.com/issues/172
